CONTRIBUTING.md

```diff
@@ -31,3 +31,24 @@ If you are adding a new file it should have a header like this:
  * limitations under the License.
  */
  ```
+
+## Dependencies
+
+- [`make`](https://www.gnu.org/software/make/)
+- [`kubebuilder`](https://github.com/kubernetes-sigs/kubebuilder) - Kms-Issuer was built using the [Kubebuilder](https://book.kubebuilder.io/) framework. See the [official documentation](https://book.kubebuilder.io/quick-start.html) to get started.
+- [`Docker`](https://www.docker.com/)
+- [`go`](https://golang.org/)
+- [`kustomize`](https://kustomize.io/)
+- [`kind`](https://kind.sigs.k8s.io/) - for testing
+- [`pre-commit`](https://pre-commit.com/)
```

Comment:
I think we need to make clear to contributors that they need to install pre-commit.